lib/action_network_rest/base.rb

```diff
@@ -11,6 +11,20 @@ def url_escape(string)
       CGI.escape(string.to_s)
     end
 
+    def object_from_response(response)
+      obj = response.body
+
+      identifiers = obj[:identifiers] || []
+      qualified_actionnetwork_id = identifiers.find do |id|
+        id.split(':').first == 'action_network'
+      end
+      if qualified_actionnetwork_id.present?
+        obj.action_network_id = qualified_actionnetwork_id.sub(/^action_network:/, '')
+      end
```

Comment:
Knowing anything about Action Network's API, I think it would be useful to have a comment explaining what's the format of the `identifiers`, but maybe that's clear from the API docs so feel free to ignore this comment.

Reply:
Sure, I'll add a comment here. The deal is that the response includes something like `identifiers: ['action_network:123-abc-456', 'other_system:789']`, so we can pull out the identifier we want and transform it to just the id part.